src/chapters/guacamole-rest-api.xml

```diff
@@ -0,0 +1,836 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<chapter xml:id="guacamole-rest-api" xmlns="http://docbook.org/ns/docbook"
+    version="5.0" xml:lang="en" xmlns:xl="http://www.w3.org/1999/xlink"
+    xmlns:xi="http://www.w3.org/2001/XInclude">
+    <title>Guacamole REST API</title>
+    <indexterm>
+        <primary>API</primary>
+        <secondary>JavaScript</secondary>
+    </indexterm>
+    <indexterm>
+        <primary>guacamole-rest-api</primary>
+    </indexterm>
+    <para>
+        There are many ways to communicate with the Guacamole protocol, but it is
+        also beneficial to understand how the API works. This allows you to have the
+        flexibility of incorporate Guacamole within your application
+        without having to worry about which language to use or what packages
+        to download.
+    </para>
+
+
+    <section xml:id="base_url">
+        <title>Base URL</title>
+        <para>There are four things you will need to set the base url before
+        you can start talking to the endpoints. </para>
+
+        <itemizedlist>
+            <listitem>
+                <para>Method</para>
+                <emphasis>
+                Protocol your Guacamole instance is using e.g. http, https
+                </emphasis>
```

Comment:
The "method" is a specific HTTP concept (GET, POST, etc.), not the protocol. You probably mean "protocol" here.